app/controllers/app/project/board.js

```diff
@@ -161,4 +161,61 @@ export default class AppProjectBoardController extends PrometheusController {
 
         return statusList;
     }
+
+    /**
+     * This function firstly filter issues based on statuses. Then those filtered issues are sorted based on there priorities, higher
+     * priority issue will be on start and low will be on last. After sorting, issues are pushed into there respective milestones.
+     *
+     * @property milestoneList
+     * @returns array
+     * @method get
+     * @public
+     */
+    @computed('milestones.issues')
+    get milestoneList() {
+        let statusList = [
```

Comment:
shouldn't status list be fetched from the system based on the project.

Reply:
Yes, but that can be our future solution. Currently we have status 7 statuses for our project _**A Prometheus**_ , which includes **Won't Fix** as well. If we fetch statuses through **API** then that will first cause an issue to rendering of lanes based on statuses. Because we have divided each lane to **2 col**, then we would require 14 cols to render all lanes. So if we'll fetch statuses from our system and if statuses related to a project are more than 6, then this will break our division of lanes in milestone.
So currently static list would be an easy solution.

Reply:
Please create an issue for it and link it here.

Reply:
http://dev.projects4.me/app/project/add12ff55acb-1902-6b9c-30d5411f7729/issue/3287 **#3287**